WireDomain/Sources/WireDomain/Synchronization/PullAllFeatureConfigsSync.swift

```diff
@@ -0,0 +1,127 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireAPI
+
+// sourcery: AutoMockable
+protocol PullAllFeatureConfigsSyncProtocol {
```

Comment:
Isn't it better to have the protocol in a separate file?
I created this PR a while ago, which separated all implementations from protocols: https://github.com/wireapp/wire-ios/pull/2374

Reply:
Yep, sorry I made these commits quite a while ago. I'll fix it

Reply:
fixed in 965a39bc3e338c231e8912b3af1218061466060a